train.py

```diff
@@ -48,15 +91,20 @@ def main():
         for data in tqdm(loader, leave=False, desc='batch'):
             # Formatted as TxBx[...]
             prv_state = None
+            mask_order = None
```

Comment:
Does `mask_order` get used anywhere?

Reply:
As I mentioned before, `mask_loss` gets the best order from previous frame(`None` for the first frame) and returns the best order for the current frame, which will be used in the next frame.

Reply:
ok.